std_msgs/Header header

float32 rfdist
float32 lfdist
float32 bdist
float32 v_relative

int32[] actions
int32[] policy
int32[] old_policy

float32 v_emg
float32 yaw_ref

bool got_new_plan

int32 emergency
string crit_check
int32 current_state
int32 lane
int32 timestep
bool request
